cni/pkg/ambient/informers.go

```diff
@@ -133,18 +133,25 @@ func (s *Server) Reconcile(input any) error {
 	event := input.(controllers.Event)
 	log := log.WithLabels("type", event.Event)
```

Comment:
I tested this on:
* kind single node: works
* GKE DPv2 (cilium): does not work. DNS is somehow partially broken and not traffic goes to ztunnel
* GKE calico (not eBPF as far as I know): does not work. outbound requests make it to ztunnel but then timeout. Requests from "not captured" -> "captured" timeout before making it to any ztunnel. Same traffic node works though
* GKE plain: same as above

So I think multi-node is probably just broken, as is Cilium

Reply:
Is this https://github.com/istio/istio/tree/experimental-ambient#supported-environments target supported platform?

Reply:
Target prior supported CNI(current stage) should be kind, calico(iptables mode)?

Reply:
I would like to support every platform if possible :-)

Reply:
I believe it's kindnet.